homeassistant/components/mvglive/sensor.py

```diff
@@ -18,44 +19,31 @@
 _LOGGER = logging.getLogger(__name__)
 
 CONF_NEXT_DEPARTURE = "nextdeparture"
-
 CONF_STATION = "station"
 CONF_DESTINATIONS = "destinations"
-CONF_DIRECTIONS = "directions"
```

Comment:
This breaks configurations where `directions` was used.

Reply:
Thats true, but the new api does not have this anymore and I doubt anyone is actually using this version atm, as it is broken. I mean I can leave it in. What is the default process for stuff like this? Is there a way of deprecating it?

Reply:
In stats there is only one single active installation using this and if it's broken and a new library doesn't support it then there is not much more to do than to make a breaking change to get this integration back on track

Reply:
yeah true, just wanted to highlight that this should be mentioned in the docs as well